### PR TITLE
Routing of components for front-end

### LINKED ISSUE #14: Set up routing between components
**User Story:**

As a developer, I want to create a routing system, so that I can easily go from one component to another.

**Acceptance Criteria:**

We can easily switch between components on button clicks

**Additional Context:**
This needs to be done so other people can start working on their part of the front-end

### PR DESCRIPTION
**Pull request by Sabatr**
## Link to Issue Number
Closes #14 

## Description
Set up a routers between pages and needed components
Reinstalled libraries which were broken previously
Added basic components for others to work on

## Checklist
- [x] All feature request A/C have been met OR the bug has been fixed
- [ ] I ran all necessary tests and they pass
- [x] I rebased upstream/master onto my working branch before opening this PR
- [x] I have named my PR something sensible
- [x] I have included the issue number above
- [x] I have assigned atleast two people to review my changes

## Other Notes
Other notes/context

State management has NOT been added yet. Will do so in the future if needed

### CODE REVIEW

File: frontend/src/booking/DetailsContainer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import changePath from "../general/helperFunctions";
+import messages from "../general/textHolder";
+
+const detailMessages = messages.details;
+const DetailsContainer = () => {
+  const history = useHistory();
+
+  return (
+    <div className={style.contentContainer}>
+      {/* Input fields go here */}
```
Comment (itsercleung) [inline frontend/src/booking/DetailsContainer.js]: Not too sure how useful this comment is

File: frontend/src/booking/TimeContainer.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { useHistory } from "react-router";
+import style from "./BookingPage.module.css";
```
Comment (itsercleung) [inline frontend/src/booking/TimeContainer.js]: Does this need a style sheet imported when its parent already has "./BookingPage.module.css"?

File: frontend/src/booking/DetailsContainer.js (hunk 1 above)
Comment (Sabatr) [reply frontend/src/booking/DetailsContainer.js]: Yeah sorry, it was more of a TODO for other members

File: frontend/src/booking/TimeContainer.js (hunk 2 above)
Comment (Sabatr) [reply frontend/src/booking/TimeContainer.js]: Hmm, I don't really see an issue with reimporting it again?

**Pull request merged**